scripts/common.py

```diff
@@ -213,10 +214,12 @@
 ACCEPTANCE_TESTS_SUITE_NAMES = [
     'blog-admin-tests/assign-roles-to-users-and-change-tag-properties',
     'blog-editor-tests/try-to-publish-a-duplicate-blog-post-and-get-blocked',
+    'logged-in-user-tests/click-all-about-oppia-footer-buttons',
     'logged-in-user-tests/click-all-buttons-on-navbar',
     'logged-in-user-tests/click-all-buttons-in-about-page',
     'logged-in-user-tests/click-all-buttons-in-about-foundation-page',
     'logged-in-user-tests/click-all-buttons-in-thanks-for-donating-page',
+    'logged-in-user-tests/click-all-links-on-get-started-page',
```

Comment:
To follow the format of "click-all-???-in/on-???", maybe change the name to "click-all-buttons-in-about-oppia-footer" and "click-all-links-in-get-started-page"? 
@seanlip, What do you think?

Reply:
I actually think you would always say "on the x page," like you would with things printed on a physical page, 

I could go either way for the navbar. I suppose it depends if it's seen as a surface or a container.

> Is the about-oppia-footer specific to that page or is it the generic footer? If so then maybe I'd say click-all-buttons-in-footer.

It is about the links that are in the section of the footer named "About Oppia"
![image](https://github.com/oppia/oppia/assets/9313126/57c86def-12d2-43f2-a135-2d52c79856ba)

Reply:
Hm, I'm OK with "on" too but let's fix everything so it's consistent (I mainly want everything to be consistent, I'm not too worried about in/on).

Re footer, ah I see. I think it would make more sense if we also had click-all-links-in-teach-learn-footer and click-all-links-in-follow-us-footer alongside it, as well -- then it'd be clearer that "About Oppia" modifies the footer.